public/app/plugins/datasource/stackdriver/datasource.ts

```diff
@@ -2,13 +2,13 @@ import { stackdriverUnitMappings } from './constants';
 import appEvents from 'app/core/app_events';
 import _ from 'lodash';
 import StackdriverMetricFindQuery from './StackdriverMetricFindQuery';
-import { StackdriverQuery, MetricDescriptor, StackdriverOptions, Filter, VariableQueryData } from './types';
+import { Filter, MetricDescriptor, StackdriverOptions, StackdriverQuery, VariableQueryData } from './types';
```

Comment:
Out of curiosity, do you have an extension in your editor that orders your imports alphabetically? In that case, let me know what it's called. :)

Reply:
SaveActions in WebStorm :)